backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+
+@Path("/validation")
+public class ValidationEndpoint {
+
+    @Inject
+    ApplicationResources appResources;
+
+    @GET
+    public Uni<Response> validate(@QueryParam("bundle") String bundleName, @QueryParam("application") String applicationName, @QueryParam("eventtype") String eventType) {
```

Comment:
```suggestion
    public Uni<Response> validate(@QueryParam("bundle") String bundleName, @QueryParam("application") String applicationName, @QueryParam("eventType") String eventType) {
```
or event shorter with the new `quarkus-resteasy-reactive` annotations:
```suggestion
    public Uni<Response> validate(@RestQuery String bundleName, @RestQuery String applicationName, @RestQuery String eventType) {
```

Reply:
done. thx